common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -268,17 +268,39 @@ export interface ISequencedDocumentMessage {
 
 	/**
 	 * Experimental field for storing the rolling hash at sequence number.
-	 * @alpha
```

Comment:
Note: it isn't valid to annotate a member of an interface as @alpha. An interface member cannot be hidden from consumers, or they would not be able to implement it.

See here for guidance on release tags: https://github.com/microsoft/FluidFramework/wiki/TSDoc-Guidelines#release-tags